tools/distpartitioning/convert_partition.py

```diff
@@ -54,14 +54,21 @@ def create_dgl_object(graph_name, num_parts, \
     dictionary
         map between edge type(string)  and edge_type_id(int)
     """
-
     #create auxiliary data structures from the schema object
-    global_nid_ranges = schema['nid']
-    global_eid_ranges = schema['eid']
-    global_nid_ranges = {key: np.array(global_nid_ranges[key]).reshape(
-        1, 2) for key in global_nid_ranges}
-    global_eid_ranges = {key: np.array(global_eid_ranges[key]).reshape(
-        1, 2) for key in global_eid_ranges}
+    node_info = schema["nid"]
```

Comment:
now `schema` is different and becomes more complex. it should be explained in the comments.

Reply:
The structure of the schema is explained in detail in the function header section for this function, as we have discussed over the chime call.